### DISCUSSION TITLE
Configurare dispositivi Matter-thread su HA

### DISCUSSION THREAD

**Discussion by frabar55** [indomus/forum#3244] [HUB: Home Assistant]
salve a tutti,
ho seguito la guida per installare un OTBR su un Nipogi dove è installato anche HA core in Docker. L'antenna è una Sonoff E con firmaware RCP NON multipan. La rete thread e relativo OTBR sono su e funzionanti ma, se cerco di collegare un dipositivo (Aqara Motion P2) la risposta è sempre la stessa: e richiesto un border router!!!
Ho letto svariati post e consigli ma non c'è niente da fare, non si riesce. Qualcuno può darmi una mano?
Grazie

**Comment by frabar55** [comment]
possibile che a nessuno interessa questo argomento? capisco che matter-thread sono ancora "sperimentali" ma bisogna pur cominciare!

**Comment by indomus** [comment]
Il problema è che OTBR implementato allo scopo che giustamente gli vuoi dedicate è molto, molto, molto ballerino. Tra l'altro, è molto difficile fare troubleshooting.

**Reply by frabar55** [reply]
grazie, quindi che strada suggerisci di percorrere?

**Reply by indomus** [reply]
prima di tutto, verificare che la configurazione sia davvero ok - inclusi i requisiti di base (rete ecc).
se non riesci, o aspetti che escano firmware e versioni di OTBR più stabili, oppure ti compri un router di frontiera di altro tipo.

**Reply by frabar55** [reply]
grazie
